bluesky_queueserver/manager/profile_ops.py

```diff
@@ -250,6 +262,148 @@ def load_profile_collection(path, *, patch_profiles=True, keep_re=False):
     return nspace
 
 
+def load_startup_module(module_name, *, keep_re=False):
+    """
+    Populate namespace by import a module.
+
+    Parameters
+    ----------
+    module_name: str
+        name of the module to import
+    keep_re: boolean
+        Indicates if ``RE`` and ``db`` defined in the module should be kept (``True``)
+        or removed (``False``).
+
+    Returns
+    -------
+    nspace: dict
+        namespace that contains objects loaded from the module.
+    """
+    importlib.invalidate_caches()
+
+    _module = importlib.import_module(module_name)
+    nspace = _module.__dict__
+
+    _discard_re_from_nspace(nspace, keep_re=keep_re)
+
+    return nspace
+
+
+class StartupLoadingError(Exception):
+    ...
+
+
+def load_startup_script(script_path, *, keep_re=False, enable_local_imports=False):
```

Comment:
A flag that is off by default is a decent compromise here.

Reply:
Since I spent time experimenting with script loading, I decided to implement all the features that I could make work so that in the future we could easily turn them on if needed. The case of `enable_local_imports=True` is covered by unit tests and seem to work fine. 

I also would like to note, that RE Worker is loading a script/module/startup collection only once. In order to reload startup code one would close RE Worker (exit or terminate the process), then start a new process and then load startup code (again only once). So I expect that a lot of issues that could come up when a script or a module is loaded multiple times within the same process would not come up here.